source/features/trending-menu-item.tsx

```diff
@@ -3,16 +3,16 @@ import features from '../libs/features';
 import {isTrending} from '../libs/page-detect';
 import {safeElementReady} from '../libs/dom-utils';
 
-async function init() {
+async function init(): Promise<boolean | void> {
 	const selectedClass = isTrending() ? 'selected' : '';
 	const issuesLink = await safeElementReady('.HeaderNavlink[href="/issues"], .header-nav-link[href="/issues"]');
 	if (!issuesLink) {
 		return false;
 	}
 
-	issuesLink.parentNode.after(
-		<li class="header-nav-item">
-			<a href="/trending" class={`js-selected-navigation-item HeaderNavlink px-lg-2 py-2 py-lg-0 ${selectedClass}`} data-hotkey="g t">Trending</a>
+	issuesLink.parentElement!.after(
```

Comment:
minor code refactor

Reply:
Yeah that's totally fine, `.parentElement` is usually what we mean